src/libos/crates/host-socket/src/stream/states/connected/send.rs

```diff
@@ -46,7 +46,9 @@ impl<A: Addr + 'static, R: Runtime> ConnectedStream<A, R> {
     fn try_sendmsg(self: &Arc<Self>, bufs: &[&[u8]], flags: SendFlags) -> Result<usize> {
         let mut inner = self.sender.inner.lock().unwrap();
 
-        if !flags.is_empty() && flags != SendFlags::MSG_DONTWAIT {
+        if !flags.is_empty()
+            && flags.intersects(!(SendFlags::MSG_DONTWAIT | SendFlags::MSG_NOSIGNAL))
+        {
             todo!("Support other flags");
```

Comment:
Can you please add the flag information in the error message "todo!("Support other flags");" to help devs know what is the required flag in the log file?